Bio/Align/substitution_matrices/__init__.py

```diff
@@ -91,7 +91,7 @@ def __new__(cls, alphabet=None, dims=None, data=None, dtype=float):
             if dims is None:
                 dims = 1
             elif dims not in (1, 2):
-                raise ValueError("dims should be 1 or 2 (found %d)" % dims)
+                raise ValueError("dims should be 1 or 2 (found %s)" % str(dims))
```

Comment:
I don't think you need the ``str(...)`` here, it is implicit by using ``%s``

Reply:
except if `dims` is a tuple

Reply:
Subtle. Thanks.